src/OrchardCore.Modules/OrchardCore.Lucene/Services/LuceneIndexingService.cs

```diff
@@ -104,7 +104,7 @@ public async Task ProcessContentItemsAsync(string indexName = default)
                 // Create a scope for the content manager
                 var shellScope = await _shellHost.GetScopeAsync(_shellSettings);
 
-                await shellScope.UsingAsync(async scope =>
+                await shellScope.UsingServiceScopeAsync(async scope =>
```

Comment:
@jtkech Let me know if we shouldn't do that. But it seems like we shouldn't throw any Tenant event from this service since it's also triggered by a background task.